xmlua/libxml2.lua

```diff
@@ -68,6 +69,17 @@ else
   libxml2.xmlFree = xml2.xmlFree
 end
 
+local function __xmlC14NDocSaveToIsAvailable()
+  local success, err = pcall(function()
+    local func = xml2.xmlC14NDocSaveTo
+  end)
+  return success, err
+end
+
+if __xmlC14NDocSaveToIsAvailable() then
```

Comment:
Do we need this check?

Reply:
AlmaLinux 8 and Ubuntu 20.04 ships c14n enabled libxml2.
We can assume that major platforms enable c14n. Let's remove this check.

Reply:
removed it.